images/linux/scripts/installers/bicep.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+################################################################################
+##  File:  bicep.sh
+##  Desc:  Installs bicep cli
+################################################################################
+
+# Fetch the latest Bicep CLI binary
+curl -Lo bicep.bin https://github.com/Azure/bicep/releases/latest/download/bicep-linux-x64
+# Mark it as executable
+chmod +x ./bicep.bin
+# Add bicep to PATH (requires admin)
+sudo mv ./bicep.bin /usr/local/bin/bicep
```

Comment:
why do we need .bin extension prior to moving the file?

Reply:
No reason it couldn't change. I wanted to stay in line with our official install script: https://github.com/Azure/bicep/blob/main/docs/installing.md#linux